src/components/Alert.ts

```diff
@@ -1,8 +1,8 @@
-import { DOM, StatelessComponent } from "react";
+import { StatelessComponent, createElement } from "react";
 
 export const Alert: StatelessComponent<{ message?: string }> = (props) =>
     props.message
-        ? DOM.div({ className: "alert alert-danger widget-time-series-alert" }, props.message)
```

Comment:
Make component generic by passing the  css class to allow it to be used for other alert types ie danger, success,